source/configure/calls-deployment.rst

```diff
@@ -0,0 +1,187 @@
+Calls self-hosted deployment
+============================
+
+This document provides information on how to successfully make the Calls plugin work on self-hosted deployments. It also outlines some of the most common deployment strategies with example diagrams.
+
+Components
+----------
+
+- **Calls plugin**: This is the main entry point and a requirement to enable channel calls. 
+- **RTCD**: This is an optional service that can be deployed to offload all the functionality and data processing involved with the WebRTC connections. This is the preferred solution for a performant and scalable deployment. With RTCD, the Mattermost server will be minimally impacted when hosting a high number of calls.
```

Comment:
I wonder if we should signal more explicitly that `rtcd` is an Enterprise only feature, maybe through a label?

Reply:
I've added a label to this point. Thanks! :)